### PR TITLE
Add Qovery as deployment option

### PR DESCRIPTION
**Pull request by arnaudjnn**
Hi Blitz team,

As discussed this week with @evoxmusic I added docs on deploying Blitz to Qovery - feedback is welcome.

We're going to add a Blitz.js template soon to speed up the setup!

Arnaud.

### CODE REVIEW

**Comment by JuanM04** [comment]
Thank you for your contribution!!

While we review your guide, could you sync your branch with `main`? I just pushed some changes: `documentation.js` was deleted in favor of `documentation.json`

File: app/core/navs/documentation.js [hunk 1]
```diff
@@ -408,6 +408,11 @@ export const documentationNav = [
       iconDarkPath: "/img/deploying-to-production-white.svg",
     },
     pages: [
+      {
+        title: "Deploy to a Server on Qovery",
+        sidebar_label: "To Qovery",
+        href: "/docs/deploy-wovery",
+      },
```
Comment (flybayer) [inline app/core/navs/documentation.js:415]: Thanks @arnaudjnn!

Please move this to between Heroku and Vercel.

File: app/pages/docs/deploy-qovery.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+3. Give a name to your PostgreSQL databse.
+4. Click on "Next".
+
+#### With storage:
+
+1. Select "Storage" among the services.
+2. Give a name to your storage.
+3. Select the storage type between Slow HDD, HDD, SSD, and Fast SSD. (SSD is recommended).
+4. Select the size.
+5. Give a mount point.
```
Comment (flybayer) [inline app/pages/docs/deploy-qovery.mdx:27]: Can you add more docs to this section? I.e. what is storage? Is it needed? What is it used for?

File: app/pages/docs/deploy-qovery.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+sidebar_label: To Qovery
+---
+
+1. Visit the [Qovery dashboard](https://console.qovery.com) to create an account if you don't already have one.
+2. Click on "Create a new project" and give a name to your project.
+3. Click on "Next".
+4. Click on "Create an application" then choose "I have an application" and select your GitHub or GitLab repository where your Blitz app is located.
+5. Click on "Next".
+6. Add a service or click "Next"
+7. Click "Deploy"
```
Comment (flybayer) [inline app/pages/docs/deploy-qovery.mdx:12]: - You need to add a step for adding the `SESSION_SECRET_KEY` environment variable
- You need to add a step for changing package.json build script to `NODE_ENV=production blitz build && blitz prisma migrate deploy --preview-feature`

Those are both in instructions for the other platforms.

**Review by flybayer — CHANGES_REQUESTED** [review]

File: app/core/navs/documentation.js (hunk 1 above)
Comment (flybayer) [reply app/core/navs/documentation.js:415]: Still need to change this

File: app/pages/docs/deploy-qovery.mdx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
++ "start:production": "blitz start --port $PORT",
++ "build": "blitz build"
+}
+```
+
+3. Create a `Procfile` file inside the root of your project with the
+   following content.
+
+```
+release: npx blitz prisma migrate deploy --preview-feature
```
Comment (flybayer) [inline app/pages/docs/deploy-qovery.mdx:37]:
```suggestion
release: npx blitz prisma migrate deploy
```

**Review by flybayer — CHANGES_REQUESTED** [review]
Sweet @arnaudjnn, still one change left

**Comment by JuanM04** [comment]
Merged. I couldn't update their branch because it was deleted

**Pull request closed without merging**